styles/styles.css

```diff
@@ -217,9 +217,113 @@ main .section.highlight {
   color: var(--midnight-blue-color);
 }
 
+.default-content-wrapper h2 {
+  margin-top: 60px;
+}
+
+.default-content-wrapper h3:nth-child(odd) {
+  float: right;
```

Comment:
shouldn't we rather use grid or flex instead of floating?

Reply:
We need to check as there is no block or parent element.

Reply:
Done